react-app-drawer-component/dist/styles.css

```diff
@@ -0,0 +1,53 @@
+.sidebar-off {
+  height: 100%;
+  width: 0px;
+  transition: 0.5s;
+  display: inline-block;
+  background-color:#ffffff;
+  padding-top: 1rem;
+  position: fixed;
+  top: 0;
+  left: 0;
+  overflow: hidden;
+}
+
+.sidebar-on {
+  height: 100%;
+  width: 250px;
+  transition: width 0.5s;
+  display: inline-block;
+  background-color:white;
+  padding-top: 1rem;
+  position: fixed;
+  top: 0;
+  left: 0;
+  overflow: hidden;
+}
```

Comment:
There is a ton of duplication here. Typically we transition things by putting the default styles (including `transition`) on one class and then add a two-class selector.

```css
.sidebar {

}

.sidebar.open {

}
```